parlai/tasks/natural_questions/build.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# Download and build the data if it does not exist.
+
+import os
+from tqdm import tqdm
+
+import parlai.core.build_data as build_data
+import parlai.utils.logging as logging
+
+DATASET_PATH_GS = 'gs://natural_questions/v1.0'
+DATASET_NAME_LOCAL = 'NaturalQuestions'
+GS_UTIL_NOT_FOUND_MESSAGE = (
+    'gsutil is required for downloading the dataset. '
+    'Please follow the installation instruction from the following link: '
+    'https://cloud.google.com/storage/docs/gsutil_install')
+
+class GSUtilsNotFound(Exception):
+    pass
+
+def _check_gsutil_available():
+    logging.info('checking whether gsutil is installed')
+    stream = os.popen('which gsutil')
```

Comment:
Isn't gsutil written in python? Maybe we can call it directly from python code and not spawn a process?

Reply:
ya would be nice if we could include that

Reply:
That's right. I agree. Working on it.